### PR TITLE
Added check for image formats for news, techer, course

### PR DESCRIPTION
**Pull request by Rysbai**

### CODE REVIEW

File: course/models.py [hunk 1]
```diff
@@ -59,6 +60,11 @@ class Meta:
         verbose_name = 'Преподователь'
         verbose_name_plural = 'Преподователи'
 
+    def clean(self):
+        image_format = self.image.name.split('.')[-1].lower()
+        if image_format not in settings.ALLOWED_IMAGE_FORMATS:
+            raise ValidationError('Пожалуйста загружите фотографии в формате: jpg, jpeg или png!')
```
Comment (dadaday) [inline course/models.py]: ЗагруЗите

File: course/models.py [hunk 2]
```diff
@@ -114,6 +120,10 @@ class Meta:
         verbose_name_plural = 'Курсы'
 
     def clean(self):
+        image_format = self.image.name.split('.')[-1].lower()
+        if image_format not in settings.ALLOWED_IMAGE_FORMATS:
+            raise ValidationError('Пожалуйста загружите фотографии в формате: jpg, jpeg или png!')
```
Comment (dadaday) [inline course/models.py]: ЗагруЗите

**Review by dadaday — CHANGES_REQUESTED** [review]

File: course/models.py [hunk 3]
```diff
@@ -59,6 +60,14 @@ class Meta:
         verbose_name = 'Преподователь'
         verbose_name_plural = 'Преподователи'
 
+    def clean(self):
+        image_format = self.image.name.split('.')[-1].lower()
+        if image_format not in settings.ALLOWED_IMAGE_FORMATS:
+            raise ValidationError('Пожалуйста загружите фотографию в формате: jpg, jpeg или png!')
+
+        if self.image.width / 960 != self.image.height / 960:
```
Comment (dadaday) [inline course/models.py]: There is no need to divide by 960, if they are on both sides. Can be rewritten just as 
`if self.image.width  != self.image.height `

However, if there is some meaning to number 960, then some explanation is needed.

File: course/models.py [hunk 4]
```diff
@@ -114,6 +123,13 @@ class Meta:
         verbose_name_plural = 'Курсы'
 
     def clean(self):
+        image_format = self.image.name.split('.')[-1].lower()
+        if image_format not in settings.ALLOWED_IMAGE_FORMATS:
+            raise ValidationError('Пожалуйста загружите фотографию в формате: jpg, jpeg или png!')
+
+        if self.image.width / 960 != self.image.height / 540:
```
Comment (dadaday) [inline course/models.py]: This probably looks better: 
`if not self.image.width / self.image.height == 16/9`

Moreover, putting `16/9` to constants is preferred. For instance:

```
from django.conf import settings
if not self.image.width / self.image.height == settings.COURSE_PHOTO_RATIO
```

**Review by dadaday — CHANGES_REQUESTED** [review]

File: course/models.py (hunk 4 above)
Comment (Rysbai) [reply course/models.py]: Wow! Cool!

File: news/models.py [hunk 5]
```diff
@@ -51,6 +53,13 @@ def compress_image(self, image):
...
     def clean(self):
+        image_format = self.image.name.split('.')[-1].lower()
+        if image_format not in settings.ALLOWED_IMAGE_FORMATS:
+            raise ValidationError('Пожалуйста загружите фотографию в формате: jpg, jpeg или png!')
+
+        if self.image.width / 960 != self.image.height / 540:
+            raise ValidationError('Пожалуйста загружите фотографию с соотношением 16X9.')
+
         if not self.title_kg and not self.title_ru:
             raise ValidationError('Пожалуйста заполните поле ЗАГОЛОВОК НОВОСТЯ хотя бы на русском или на кургызском.')
```
Comment (dadaday) [inline news/models.py]: заголовок новост`И`
к`ы`ргызском

**Review by dadaday — CHANGES_REQUESTED** [review]

**Review by dadaday — APPROVED** [review]

**Pull request merged**